src/components/views/rooms/MessageComposerInput.js

```diff
@@ -541,7 +541,7 @@ export default class MessageComposerInput extends React.Component {
         let sendTextFn = this.client.sendTextMessage;
 
         if (contentText.startsWith('/me')) {
-            contentText = contentText.replace('/me', '');
+            contentText = contentText.replace('/me ', '');
             // bit of a hack, but the alternative would be quite complicated
             if (contentHTML) contentHTML = contentHTML.replace('/me', '');
```

Comment:
Oh I see I missed this one, I'm not sure if it's needed for the HTML though, since I think that HTML collapses multiple spaces.